docs/advanced/analytics/plugins.mdx

```diff
@@ -305,11 +305,11 @@ Configuration example in `src/config/analytics.js`
 {
   name: "google-tag-manager",
   needConsent: true,
-  settings: (authorization, otherAuthorizations) => {
+  settings: (authorization, otherAuthorizations) => ({
     containerId: "GTM-123xyz",
     // the userConsents option is a specific key that the plugin will use and expose in the GTM dataLayer
     userConsents: otherAuthorizations
-  }
+  }),
   script: () => import("@segment/analytics.js-integration-google-tag-manager"),
```

Comment:
As mentioned in [our thread](https://frontcommerce-hq.slack.com/archives/C03C2T9ADLY/p1669147452727279?thread_ts=1669139805.112639&cid=C03C2T9ADLY), this should be from get analytics.

```suggestion
  script: () => import("@analytics/google-tag-manager"),
```